src/zino/api/legacy.py

```diff
@@ -22,12 +25,15 @@ def requires_authentication(func: Callable) -> Callable:
 class Zino1BaseServerProtocol(asyncio.Protocol):
     """Base implementation of the Zino 1 protocol, with a basic command dispatcher for subclasses to utilize."""
 
-    def __init__(self):
+    def __init__(self, secrets_file: Optional[Union[Path, str]] = "secrets"):
```

Comment:
When there is a config-file the location of the "user database" needs to be configurable.